packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -471,6 +473,15 @@ class SizeTransition extends AnimatedWidget {
   /// A value of 0.0 (the default) indicates the center for either [axis] value.
   final double axisAlignment;
 
+  /// The factor to multiply the cross axis size of the child by.
```

Comment:
Minor clarification:
```suggestion
  /// The factor by which to multiply the cross axis size of the child.
```

Reply:
I applied the change in [doc: Better phrasing in comment](https://github.com/flutter/flutter/pull/134659/commits/076706115ca89a6a0e84e6c64d9be0df7a0b4b30)